checkov/common/bridgecrew/run_metadata/abstract_run_metadata_extractor.py

```diff
@@ -0,0 +1,83 @@
+from abc import abstractmethod
+from checkov.common.bridgecrew.run_metadata.registry import registry
+from checkov.common.bridgecrew.run_metadata.ci_variables import (
+    BC_TO_BRANCH,
+    BC_PR_ID,
+    BC_PR_URL,
+    BC_COMMIT_HASH,
+    BC_COMMIT_URL,
+    BC_AUTHOR_NAME,
+    BC_AUTHOR_URL,
+    BC_RUN_ID,
+    BC_RUN_URL,
+    BC_REPOSITORY_URL,
+    BC_FROM_BRANCH,
+)
+
+
+class RunMetaDataExtractor:
+    from_branch = ""
+    to_branch = ""
+    pr_id = ""
+    pr_url = ""
+    commit_hash = ""
+    commit_url = ""
+    author_name = ""
+    author_url = ""
+    run_id = ""
+    run_url = ""
+    repository_url = ""
+
+    def __init__(self, from_branch,
+                 to_branch,
+                 pr_id,
+                 pr_url,
+                 commit_hash,
+                 commit_url,
+                 author_name,
+                 author_url,
+                 run_id,
+                 run_url,
+                 repository_url):
+        if self.is_current_ci():
+            self.from_branch = from_branch
+            self.to_branch = to_branch
+            self.pr_id = pr_id
+            self.pr_url = pr_url
+            self.commit_hash = commit_hash
+            self.commit_url = commit_url
+            self.author_name = author_name
+            self.author_url = author_url
+            self.run_id = run_id
+            self.run_url = run_url
+            self.repository_url = repository_url
```

Comment:
```suggestion
        self.from_branch = from_branch
        self.to_branch = to_branch
        self.pr_id = pr_id
        self.pr_url = pr_url
        self.commit_hash = commit_hash
        self.commit_url = commit_url
        self.author_name = author_name
        self.author_url = author_url
        self.run_id = run_id
        self.run_url = run_url
        self.repository_url = repository_url
```
looking at the later usage, I don't see a need to do the if condition here, just set all the attributes and later you decide via `is_current_ci()` to access the correct one